sycl/test/format.py

```diff
@@ -5,6 +5,8 @@
 import os
 import re
 
+SUFFIXES = {'.hpp'}
```

Comment:
The linter is complaining about my use of single quotes.

![image](https://github.com/user-attachments/assets/f444e1d0-0ce3-4ab2-b315-c45ab6fc76a6)

Seems a bit aggressive, but I'll update

Reply:
Oh, python linter is very annoying for sure